### ISSUE TITLE
Show lengend and imprecision in map view

### ISSUE THREAD

**Issue by jholetschek** [gbif/hosted-portals#265]
I guess this is a tricky one, but I don't want it to get lost: Many participants of our workshop mentioned that they would like to see the coordinate imprecision for individual records. Probably this could be done only for large zoom levels, and I'm not sure how this would look like.

Also, for many users it wasn't obvious that the color of the dots represents the number of occurrences in the cluster - so adding a small legend on what the colors mean would be nice.

**Comment by ymgan** [comment]
+1 for this, thank you!!

**Comment by MortenHofft** [comment]
I do not believe we will be able to show uncertainties and foot print WKTs in the occurrence search map. But we should and could show them for individual occurrences. At least that is the conclusion and implementation we have on GBIF.org

**Comment by ymgan** [comment]
This is awesome Morten!! Thank you so much!!
